core/services/job/orm.go

```diff
@@ -705,61 +705,41 @@ func (o *orm) FindJobs(offset, limit int) (jobs []Job, count int, err error) {
 			return err
 		}
 		for i := range jobs {
-			err = multierr.Combine(err, o.LoadEnvConfigVars(&jobs[i]))
+			err = multierr.Combine(err, o.LoadConfigVars(&jobs[i]))
 		}
 		return nil
 	})
 	return jobs, int(count), err
 }
 
-func (o *orm) LoadEnvConfigVars(jb *Job) error {
+func (o *orm) LoadConfigVars(jb *Job) error {
```

Comment:
Does it make sense to remove this completely from ORM? This is also called in the delegate so all the functionality should be fine. IIUC only gql endpoint would be affected if this is removed.